### ISSUE TITLE
Write setup guides for user groups

### ISSUE THREAD

**Issue by RumovZ** [bohning/usdb_syncer#35]
These should cover if / how to incorporate existing collections, keep in sync with changes, create new songs etc.

**Comment by Baklap4** [comment]
@RumovZ Where to write this part? i guess the wiki can be the place?

**Comment by RumovZ** [comment]
Correct. Just please be aware that things are still changing a lot currently and documentation could become outdated quickly.
